server/models.py

```diff
@@ -1643,3 +1643,71 @@ class CanvasAssignment(Model):
     @property
     def url(self):
         return '{}/assignments/{}'.format(self.canvas_course.url, self.external_id)
+
+#########
+# Files #
+#########
+
+class ExternalFile(Model):
+
+    id = db.Column(db.Integer, primary_key=True)
+    # Bucket/Folder that the file is stored in
+    container = db.Column(db.String(255), nullable=False)
+    filename = db.Column(db.String(1024), nullable=False)
```

Comment:
Why `filename` over `file_name`? 

- http://english.stackexchange.com/a/5374/10058 
- The real reason: Python seems to go with `filename` when possible